src/bootstrap/build/util.rs

```diff
@@ -16,6 +16,26 @@ use std::process::Command;
 use bootstrap::{dylib_path, dylib_path_var};
 use filetime::FileTime;
 
+#[cfg(not(windows))]
+pub fn build_path(path: &str) -> PathBuf {
+    PathBuf::from(path)
+}
+
+#[cfg(windows)]
+pub fn build_path(path: &str) -> PathBuf {
```

Comment:
I don't think we can stomach all of these process spawns on *every invocation of the build system*, so this needs to be handled in some method.